### ISSUE TITLE
Evaluate loop list item type options

### ISSUE THREAD

**Issue by rebeccawalsh** [department-of-veterans-affairs/va.gov-team#16152]
## Issue Description
Each of the loop lists in the form have user-dictated labels for values submitted. We need to decide if these should be pure inputs, auto-suggest inputs, or dropdown selects. This includes:
- C2P1 - Add a payroll deduction
- C2P2 - Type of income
- C3P1 - Type of real estate
- C3P1 - Type of recreational vehicle
- C3P1 - Type of vehicle
- C3P1 - Vehicle make
- C3P1 - Vehicle model
- C3P1 - Type of asset
- C4P1 - Type of utility
- C4P1 - Type of living expense

---
## Tasks
- [x] Evaluate input type directions (Becca + Riley)
- [ ] (If not plain inputs) Define options or groups (Susan)
- [ ] Document decisions and attach to ticket (Susan)

## Acceptance Criteria
- [ ] Decisions attached to ticket

**Comment by 3obryandogs** [comment]
C2P1 - Add a payroll deduction - the list is fairly short
C2P2 - Type of income - is this the additional income? 
C3P1 - Type of real estate - I'm not clear on what this would include
C3P1 - Type of recreational vehicle - Can we change the wording? Recreational vehicle is a specific type of vehicle (RV)
C3P1 - Type of vehicle - the list is fairly short
C3P1 - Vehicle make - too long to include specifics; perhaps list an example like "Ford"
C3P1 - Vehicle model - too long to include specifics, perhaps list an example like "F150"
C3P1 - Type of asset - perhaps listing suggestions, but not specifics
C4P1 - Type of utility - the list is fairly short
C4P1 - Type of living expense - perhaps listing suggestions, but not specifics

**Comment by rebeccawalsh** [comment]
Hey, @3obryandogs!

What we're looking for this ticket are the list items inside our "type ahead" component. Here's an example of the type ahead component on the 526-ez form:
![image](https://user-images.githubusercontent.com/6361930/99685090-33382580-2a50-11eb-891a-503fff865033.png)

The way it works is that you can type into the field and it will suggest options in the dropdown to populate the field. If what they're looking for isn't in the dropdown, it's okay since they can type whatever they want and it will be accepted - think URL bars.

Plain text will be a good format to deliver this in, ex:
**Add a payroll deduction**
Item 1
Item 2
Item 3

Let me know if you have any questions. (RV can be changed)

**Comment by 3obryandogs** [comment]
Becca,
Gotcha! Do you have a preference for priority? This one, or the error messages?

Thanks!

**Comment by rebeccawalsh** [comment]
From what I'm seeing, this ticket is in the current sprint, while the error messaging tickets are currently in backlog and ready, which would mean this is prioritized. Not sure if error messaging was meant to be in the sprint for you. I'd suggest doing this before error messaging.

**Comment by 3obryandogs** [comment]
Working on it now.

**Comment by 3obryandogs** [comment]
Question: If the Veteran also has business expenses, are those applicable to the FSR? Or does the FSR only take in consideration “personal” expenses and assets?

C2P1 - Add a payroll deduction
•	Federal tax
•	Social Security
•	Medicare
•	State tax
•	Other tax
•	Other deductions (Note: Pay stubs may add insurance, 401K, or retirement/pension, into one category)
C2P2 - Type of “other” income
•	Rental property income
•	Part-time employment
•	Contract employment
•	Retirement payments
•	Non-VA pension payments
•	Unemployment payments
•	Leased acreage or land
C3P1 - Type of real estate
•	House(s)
•	Mobile home
•	Business property
•	Leased acreage or land
C3P1 - Type of recreational vehicle
•	Recreational vehicle (RV)
•	Boat
•	Camper
•	All-terrain vehicle (ATV or “four-wheeler”)
•	Snow mobile
C3P1 - Type of vehicle
•	Car
•	Truck
•	Motorcycle
C3P1 - Vehicle make
C3P1 - Vehicle model
C3P1 - Type of asset - You can find these items listed on your yearly income tax filings.
•	Cash (includes checking account and funds on hand)
•	Savings account
•	Savings bonds
•	Real estate 
•	Stocks and bonds
•	Oil or mineral rights or leases
•	Savings plans (401K, pension)
•	Jewelry (based on appraised value)
•	Antiques (based on appraised value)
•	Stamp, coin or art collection(s) (based on appraised value)
C4P1 - Type of utility
•	Electricity
•	Gas
•	Water
•	Sewer
•	Coal
•	Propane
C4P1 - Type of living expense – Monthly averages
•	Groceries 
•	Credit card(s)
•	Home-related services
o	Rent or house payment(s)
o	Lawn service
o	Home Owner Association fees
o	Storage unit(s)
•	Vehicle payments 
o	Car
o	Truck
o	Motorcycle
o	Recreational vehicle (RV)
o	Boat
o	Camper
o	All-terrain vehicle (ATV)
o	Snow mobile
•	Care-related services
o	Child care
o	Elder care
o	Disability care
o	Nursing home
•	Education-related services
o	School tuition
o	School meals
o	Special needs education programs
•	Insurance
o	Home(s) insurance
o	Vehicle(s) insurance
o	Renter’s insurance
o	Medical insurance
	Medicare
	Medicare supplemental benefits
o	Dental insurance
o	Vision insurance
o	Life insurance
o	Supplemental insurance
•	Health care-related services
o	Hospital 
o	Prescriptions 
o	Home health
o	Special needs health care programs
•	Communication-related services
o	Internet
o	Cable or satellite
o	Cell phone service
o	Landline phone service
•	Transportation-related services
o	Bus
o	Train
o	Subway
o	Parking
o	Toll(s)
o	Ferry

**Comment by rileyorr** [comment]
Hey, @3obryandogs! Thanks for providing this list. I put your suggested content [in this Google Doc](https://docs.google.com/document/d/1P6GJyMbtoBN_ythEQoCSvVGh9QjGlk1BGhB0spjdMVU/edit?usp=sharing); Becca and I added some questions/comments for you to take a look at!

**Comment by 3obryandogs** [comment]
Good morning, y’all! Thank you!

**Comment by 3obryandogs** [comment]
I’ve added my comments to the document.

**Comment by rebeccawalsh** [comment]
@3obryandogs Great! So now that you've got a better idea of what we're looking for this piece of work, please delve into each of the sections to make sure we're being as comprehensive as possible, using the right language where things get a little dicey (special needs for example.) Remember that this is a type-ahead list, so we can have many items in each input. The categories won't be displayed (see the screenshot above for an example of how the component will look.) We're not needing this content immediately, so more extensive research should be possible. Thank you!

**Comment by 3obryandogs** [comment]
Riley and Becca, the list now is quite long. I’ve broken out every category. We may want to be consistent with wording, such as credit card payment, doctor bills. Your thoughts?

**Comment by rileyorr** [comment]
Hi, @3obryandogs! It's okay that the lists are long - as users type into the field it'll filter down the options based on the characters inputted. I made a [UXPin prototype](https://preview.uxpin.com/fa323c51a84318fe2c5aec9cf6f125fa4062625b#/pages/134282568/comment/sitemap?mode=i) showing the filtering that may give you an idea of what our goal is. 

I added a few more comments and resolved some of your proposed updates! I think this work may bleed into the beginning of next sprint, but if you think the list is comprehensive it might be a good idea to put it in a digestible format for DMC and/or Peggy to review. I trust your judgment for whichever next step you think is appropriate. LMK if you have questions!
